library/core/src/fmt/mod.rs

```diff
@@ -564,7 +564,7 @@ impl Display for Arguments<'_> {
     on(
         crate_local,
         label = "`{Self}` cannot be formatted using `{{:?}}`",
-        note = "add `#[derive(Debug)]` or manually implement `{Debug}`"
+        note = "add `#[derive(Debug)]` to `{Self}` or manually implement `{Debug}` for `{Self}`"
```

Comment:
This sounds a bit verbose, I like @estebank's wording:

```suggestion
        note = "add `#[derive(Debug)]` or manually `impl {Debug} for {Self}`"
```

Reply:
@JohnTitor How would you feel about keeping the to Self in the first half, and only making the change in the second half? I'm a bit concerned that having to read to the end to get the `for {Self}` component may make the message less readable.

Reply:
Hmm yeah, makes sense. It's fine to me.

Reply:
Thanks!